### DISCUSSION TITLE
Entire log repeating every ten seconds

### DISCUSSION THREAD

**Discussion by thomcom** [argoproj/argo-cd#19732] [Q&A]
Hi, I'm running Argocd v2.11.7+e4a0246 and seeing repeat log statements across all the pods in my cluster. If I bring up the logs for a particular pod, it's entire logging history gets repeated in the log window every ten seconds, instead of just appending the new log messages to the existing history every ten seconds. It is annoying and I'd love to fix it, but have no idea how. Any suggestions?

**Comment by pasha-codefresh** [comment]
Could you please add details? some screenshot maybe, this issue looks really weird

**Reply by dank303** [reply]
I've just seen the same thing happen, but it seems to be about every 30 seconds.   If I download or copy the log using the on-screen buttons, I get a single copy (nothing repeated).  But the log display in argocd gets the entire log repeated.   I've only seen this in the one pod I'm working with, but haven't looked at others lately.

This is version v2.13.1+af54ef8

**Reply by Sjalu** [reply]
Hey,

I'm having the same issue as well - been like it for as long as I can remember using ArgoCD. Gotten used to it, but it's really annoying.

You can see from timestamps in attached screenshot how it repeats itself.

<img width="2038" height="1201" alt="image" src="https://github.com/user-attachments/assets/2574befd-bce3-4672-a7dd-ccc50ed3078b" />

**Comment by mbench777** [comment]
We got the same problem, the same logs were repeating every 15s in ArgoCD UI and the root cause was the ingress timeout cutting the gRPC stream. We used contour which had the default 15s timeout of envoy, we could see in argocd-server logs "finished streaming call" every 15s. We increased the timeout in the ingress and the issue was solved.
